### DISCUSSION TITLE
Enable Kerberos support for Kafka auth

### DISCUSSION THREAD

**Discussion by trickster** [redpanda-data/connect#876] [Ideas]
Vector has complete librdkafka config support which enables Kerberos auth. Would like to have a similar feature or just Kerberos support in general for sources/sinks.

**Comment by Jeffail** [comment]
Hey @trickster, needs a bit of investigation but I think sarama might support kerberos now so this could be fairly quick to add, I've raised an issue https://github.com/Jeffail/benthos/issues/877.
